src/components/VisuallyHidden.tsx

```diff
@@ -1,43 +1,18 @@
 import React from "react"
-import styled from "@emotion/styled"
+import { VisuallyHidden as VisuallyHiddenSpan } from "@chakra-ui/react"
```

Comment:
Alias here as conflicted with component name

Reply:
One thing on this, can you call it `ChakraVisuallyHidden`? since we are calling those aliases with the `Chakra` prefix in other files